src/schedule/dto/update-schedule.dto.ts

```diff
@@ -1,4 +1,6 @@
 import { PartialType } from '@nestjs/mapped-types';
 import { CreateScheduleDto } from './create-schedule.dto';
 
-export class UpdateScheduleDto extends PartialType(CreateScheduleDto) {}
+export class UpdateScheduleDto extends PartialType(CreateScheduleDto) {
+  lessons: [];
```

Comment:
Wyczytałem w dokumentacji że tutaj klasa nie musi mieć żadnych parametrów, bo samo z siebie to wynika jak rozszerzamy ten PartialType